lib/pages/movie_details_page.dart

```diff
@@ -0,0 +1,139 @@
+                  StringConstants.movieDetailDate +
+                  result.releaseDate +
+                  StringConstants.movieDetailRating +
+                  result.voteAverage.toString(),
+              textAlign: TextAlign.start,
+              style: TextStyle(
+                color: Colors.white,
+                wordSpacing: MeasuresConstants.movieDetailInfoWordSpacing,
+                fontSize: MeasuresConstants.movieDetailInfoFontSize,
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(
+              left: MeasuresConstants.playButtonLeftRightPadding,
+              right: MeasuresConstants.playButtonLeftRightPadding,
+            ),
+            child: ElevatedButton(
+              style: ButtonStyle(
+                backgroundColor: MaterialStateProperty.resolveWith(
+                  (
+                    states,
+                  ) =>
+                      Colors.white,
+                ),
+              ),
+              onPressed: () {},
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Icon(
+                    Icons.play_arrow,
+                    color: Colors.black,
+                  ),
+                  Text(
+                    StringConstants.playButtonText,
+                    style: TextStyle(
+                      color: Colors.black,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(
+              top: MeasuresConstants.movieDetailOverviewTopPadding,
+              left: MeasuresConstants.movieDetailOverviewLeftRightBottomPadding,
+              right:
+                  MeasuresConstants.movieDetailOverviewLeftRightBottomPadding,
+              bottom:
+                  MeasuresConstants.movieDetailOverviewLeftRightBottomPadding,
+            ),
+            child: Text(
+              result.overview,
+              style: TextStyle(
+                color: Colors.white,
+                fontSize: MeasuresConstants.movieDetailOverviewFontSize,
+              ),
+            ),
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Icon(
+                Icons.share_outlined,
+                color: Colors.white,
+              ),
+              SizedBox(
+                width: MeasuresConstants.movieDetailSixedBoxWidth,
+              ),
+              Icon(
+                Icons.favorite_border,
+                color: Colors.white,
+              ),
+              SizedBox(
+                width: MeasuresConstants.movieDetailSixedBoxWidth,
+              ),
+              Icon(
+                Icons.add,
+                color: Colors.white,
+              ),
+            ],
+          )
+        ],
+      ),
+    );
+  }
+}
```

Comment:
Can you separate this into different widgets like: BackdropImage, Overview, Movie, some widget that has language, date and rating.

Reply:
Add the last one that contains Icons of the bottom.

Reply:
Done.